docker-compose-configuration/.env

```diff
@@ -0,0 +1,18 @@
+#### Project/deployment name ####
+# needs to be unique i.e. will be used as container and network name as well as for the vhost entry
+COMPOSE_PROJECT_NAME=rds-local
+
+# can be used to pass specific platform parameter that will override any parameter set in the config files
+# to use a different log configuration
+#PLATFORM_OPTS=-Dlog4j.configurationFile=file:///var/lib/jetty/webapps/etc/log4j2-debug.xml
+PLATFORM_OPTS=
+
+#### DOCKER IMAGE VERSIONS ####
+
+BLAZEGRAPH_IMAGE=metaphacts/blazegraph-basic:2.2.0-20160908.003514-6-jetty9.4.31-jre8-268ccbf
+PLATFORM_IMAGE=metaphacts/metaphacts-platform:3.6
```

Comment:
this version does not exist, the correct name is `metaphacts/glam-community:3.6.0` (also note the `.0` at the end!)

Reply:
I'll change the version, but I am sure it exists. We used exactly this version for RDS.

Reply:
metaphacts-platform is private, please change to glam-community

Reply:
done.